package/gluon-status-page/files/lib/gluon/status-page/www/cgi-bin/dyn/neighbours-nodeinfo

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+
+echo 'Access-Control-Allow-Origin: *'
+
+batctl if | grep -q "^$QUERY_STRING:"  || exit 1
```

Comment:
Using the query string as a grep argument without `-F` seems like very bad idea.